thrift/src/main/java/com/linecorp/armeria/common/thrift/text/StructContext.java

```diff
@@ -132,63 +147,13 @@ protected TField getTFieldByName(String name) throws TException {
      * TProtocol.writeStructBegin(), rather than relying on the stack trace.
      */
```

Comment:
So this description goes to `DefaultCurrentThriftMessageClassFinder` right?

Reply:
`StackWalking` and `getStackTrace`  are getting the same way on the stacktrace ? [JEP-259](https://openjdk.java.net/jeps/259)

Reply:
Ah yes. Didn't read it carefully. Forget about the comment. 😉